src/snyk/snykOss/editor/editorDecorator.ts

```diff
@@ -58,7 +57,7 @@ export class EditorDecorator {
           module.line - 1,
           this.editorLastCharacterIndex,
         ),
-        renderOptions: getRenderOptions(messages.vulnerabilityCount.fetchingVulnerabilities, this.themeColorAdapter),
+        renderOptions: getRenderOptions('', this.themeColorAdapter),
```

Comment:
question: Why did you remove the message?

Reply:
the change to the `this.workspace.onDidChangeTextDocument` workspace listener meant that this message did not update; as it's only briefly visible, I thought to leave it blank.

This can be reverted once the `this.workspace.onDidChangeTextDocument` listener is fixed.